otterdog/eclipse-windowbuilder.jsonnet

```diff
@@ -82,5 +82,15 @@ orgs.newOrg('eclipse-windowbuilder') {
         enabled: false,
       },
     },
+    orgs.newRepo('windowbuilder-images') {
+      allow_merge_commit: true,
+      default_branch: "master",
```

Comment:
you really want to have a master branch in a new repo? The default is now main, so if you leave out this option, the default will be used.

Reply:
The branches in the other repos are also called "master". Having it be "master" for some but "main" for others looks a little weird to me.

Reply:
as you prefer ofc